src/pages/Contributing/GitConventions/index.md

```diff
@@ -1,4 +1,113 @@
 ---
 title: 'Git conventions'
-order: 10
----
\ No newline at end of file
+order: 5
+---
+
+## Create and name a new branch
+
+Each branch name consists of a `type`, an optionnal issue `id` and a `description` (_Definitions are available further down the page_)
```

Comment:
- remove (_Definitions are available further down the page_)
- and just say that the type as to match the commit type 
- optionnal issue `id` coming from a Jira ticket or a GitHub issue (with link on both)